packages/connectivity/src/scp-cf/identity-service.ts

```diff
@@ -13,7 +16,8 @@ export async function exchangeToken(
   options: DestinationOptions
 ): Promise<string> {
   const xsuaaService = getXsuaaService({
-    disableCache: !options.cacheVerificationKeys
+    disableCache: !options.cacheVerificationKeys,
+    credentials: getDestinationServiceCredentials()
```

Comment:
because we are retrieving this token only for the destination service, so this should be enough. Honestly, we need to clarify, whether there are cases where we would need to do this for connectivity. I am even thinking splitting this off this PR.